x-pack/legacy/plugins/lens/public/xy_visualization_plugin/xy_suggestions.ts

```diff
@@ -146,30 +175,68 @@ function getSuggestion(
     return buildSuggestion(options);
   }
 
+  const sameStateSuggestions: Array<VisualizationSuggestion<State>> = [];
+
   // if current state is using the same data, suggest same chart with different presentational configuration
 
   if (xValue.operation.scale === 'ordinal') {
     // flip between horizontal/vertical for ordinal scales
-    return buildSuggestion({
-      ...options,
-      title: i18n.translate('xpack.lens.xySuggestions.flipTitle', { defaultMessage: 'Flip' }),
-      isHorizontal: !options.isHorizontal,
-    });
+    sameStateSuggestions.push(
+      buildSuggestion({
+        ...options,
+        title: i18n.translate('xpack.lens.xySuggestions.flipTitle', { defaultMessage: 'Flip' }),
+        isHorizontal: !options.isHorizontal,
```

Comment:
In the near future we want to remove the ability to have horizontal line/area charts, only horizontal bars will be allowed. Not something for this PR though